### PR TITLE
Playing status and resurrect tests

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

Status should switch to play even when loading, including when skipping tracks.
Also react to play when ready status changes, so pause is immediate.

#### WHY

Better experience for users.

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: media-data/src/main/java/com/google/android/horologist/media/data/PlayerRepositoryImpl.kt [hunk 1]
```diff
@@ -129,7 +130,7 @@ public class PlayerRepositoryImpl : PlayerRepository, Closeable {
      * [Player.getPlaybackState] properties.
      */
     private fun updateState(player: Player) {
-        _currentState.value = if (player.isPlaying) {
+        _currentState.value = if (player.isPlaying || player.isLoading && player.playWhenReady) {
```
Comment (luizgrp) [inline media-data/src/main/java/com/google/android/horologist/media/data/PlayerRepositoryImpl.kt:133]: nit: could you please update the kdocs to either mention this new property that is being used or just make it more generic so it doesn't get outdated quickly again

**Review by luizgrp — APPROVED** [review]

File: .gitattributes [hunk 2]
```diff
@@ -1,2 +1,4 @@
 **/test/snapshots/** filter=lfs diff=lfs merge=lfs -text
 **/figma/** filter=lfs diff=lfs merge=lfs -text
+media-data/src/test/assets/media/mp4/testvid_1022ms.mp4 filter=lfs diff=lfs merge=lfs -text
+media3-backend/src/test/assets/media/mp4/testvid_1022ms.mp4 filter=lfs diff=lfs merge=lfs -text
```
[the remark this answers is not in the document]
Comment (yschimke) [reply .gitattributes:4]: Not deleted, moved to LFS.

**Review by luizgrp — APPROVED** [review]

File: .gitattributes [hunk 3]
```diff
@@ -1,8 +1,6 @@
 **/test/snapshots/** filter=lfs diff=lfs merge=lfs -text
 **/figma/** filter=lfs diff=lfs merge=lfs -text
-media-sample/src/main/res/drawable-mdpi/album_art.jpg filter=lfs diff=lfs merge=lfs -text
-media-sample/src/main/res/drawable-mdpi/album_art_2.jpg filter=lfs diff=lfs merge=lfs -text
-media-sample/src/main/res/drawable-mdpi/album_art_3.jpg filter=lfs diff=lfs merge=lfs -text
-media-sample/src/main/res/drawable-mdpi/kai.jpg filter=lfs diff=lfs merge=lfs -text
-media-sample/src/main/res/drawable-mdpi/kyoto.jpg filter=lfs diff=lfs merge=lfs -text
+media-data/src/test/assets/media/mp4/testvid_1022ms.mp4 filter=lfs diff=lfs merge=lfs -text
+media3-backend/src/test/assets/media/mp4/testvid_1022ms.mp4 filter=lfs diff=lfs merge=lfs -text
+media-sample/**/drawable-mdpi/*.jpg filter=lfs diff=lfs merge=lfs -text
```
[the remark this answers is not in the document]
Comment (yschimke) [reply .gitattributes:5]: Why, my mental model of lfs is something that resolves these files from metadata to real files on checkout.

[the remark this answers is not in the document]
Comment (luizgrp) [reply .gitattributes:5]: my concern comes from my ignorance of how lfs works 😅 if that's the case then that's fine 👍

**Pull request merged**